spec/components/provider_interface/interview_and_course_summary_component_spec.rb

```diff
@@ -16,14 +16,24 @@
     expect(component).to include(interview.application_choice.course.provider.name)
   end
 
-  it 'displays the course name' do
-    expect(component).to include(interview.application_choice.course.name)
+  it 'displays the course name and code' do
+    expect(component).to include(interview.application_choice.course.name_and_code)
   end
 
   it 'displays interview location' do
     expect(component).to include(interview.location)
   end
 
+  it 'displays additional_details' do
+    expect(component).to include(interview.additional_details)
+  end
+
+  it 'displays additional details as None when no additional details provided' do
+    interview = create(:interview, additional_details: '')
+    component = render_inline(described_class.new(interview: interview, user_can_change_interview: true)).text
```

Comment:
You should be able to test this change by using the interview object already available in this spec and simply updating it in your example using `interview.additional_details = ''`